docs/products/flink/howto/real_time_alerting_solution.rst

```diff
@@ -221,7 +221,7 @@ This setup uses host-specific thresholds that are stored in PostgreSQL as a basi
       :lines: 57-59
       :language: sql
 
-#. Select your Kafka service, enter ``CPU_OUT_FILTER_PG`` as the name, ``cpu_load_stats_real_filter_pg`` as the topic, and the following as the SQL schema, then click **Create Table**:
+#. Select your Kafka service, use the default selection for connector type, key, and value data format, enter ``CPU_OUT_FILTER_PG`` as the name, ``cpu_load_stats_real_filter_pg`` as the topic, and the following as the SQL schema, then click **Create Table**:
```

Comment:
"use the default selection for connector type" <- I'm not sure which is the default selection, could be mention the "standard Kafka connector type" and JSON for key and value data format?